src/dnx.coreclr.unix/tpa.cpp

```diff
@@ -1,97 +1,85 @@
 // Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Yes. It must have been missed when we updated headers since tpa.cpp files are auto generated.